apps/src/codebridge/FileBrowser/styles/fileBrowser.css

```diff
@@ -28,16 +28,31 @@
 .files-toolbar {
   display: flex;
   justify-content: space-between;
-  font-size: 75%;
+  color: white;
   padding: 5px;
+  background-color: #4f555c;
 }
 
 .file-browser ul li span.button-bar {
   display: flex;
-  justify-content: flex-end;
+  justify-content: flex-start;
+  flex-direction: column;
   cursor: pointer;
 }
 
+.file-browser ul li span.button-bar span:hover {
+  background-color: #54595e;
```

Comment:
can we use color constants here instead (from color.scss, would require changing the file to scss)? Ideally this file should be a css module but that can be a follow up.

Reply:
Done!
Also moved it into a scss module.